### DISCUSSION TITLE
Node type without handles

### DISCUSSION THREAD

**Discussion by e-gloo** [bcakmakoglu/vue-flow#902] [Q&A] [answered]
Hi, it would be great if we could create node without handles.
The main use case would be with parent and children nodes, the parent node doesn't always need to be connected. So if we could have a node without handle that would work like a group that has children, and when dragging the group all children are dragged along.
Another use case would be when we set `connectable` to false.

**Comment by bcakmakoglu** [comment, marked as answer]
You can just create a custom node without handles 😅 
This is not really something that has to be explicitly provided

**Comment by e-gloo** [comment]
Yeah I figured a bit later that I could do it with a custom node :sweat_smile: 
It's just a bit confusing because in the node properties, we can specify the type of node which will set number of handles, and we can specify the position of them, but there is no value option to not include them.
But anyway, I managed to do it, sorry for taking your time and tank you for your answer!
